data/data/bootstrap/files/usr/local/bin/release-image-download.sh.template

```diff
@@ -17,3 +17,20 @@ do
     echo "Pull failed. Retrying $RELEASE_IMAGE..."
 done
 
+{{if .IsOKD -}}
+# Pivot bootstrap to FCOS + OKD machine-os-content
+if [ ! -f /opt/openshift/.pivot-done ]; then
```

Comment:
Could maybe check for the presence of `/usr/bin/crio` too as a sanity check?

Reply:
I guess we could place a special file (in `okd-machine-os` repo) which means the machine has pivoted. `.pivot-done` is a bit unreliable - file is being created when "machine-config-daemon pivot" is about to be called, instead of after, as it also does the machine reboot.